tests/test_issue421_cross_session_scope.py

```
@@ -0,0 +1,52 @@
+        session_b = beam.BeamMemory(db_path=db_path, session_id="session-b")
+        results = session_b.recall("issue 421 cross session memory")
+
+        assert any("issue 421 cross session memory" in r["content"] for r in results)
+    finally:
+        _reload_beam_with_cross_session(False)
+
+
+def test_cli_store_honors_default_scope_global(monkeypatch, tmp_path, capsys):
+    monkeypatch.setenv("MNEMOSYNE_DEFAULT_SCOPE", "global")
+
+    import mnemosyne.cli as cli
```

Ruff (0.15.20)

[warning] Use `from mnemosyne import cli` in lieu of alias

Replace with `from mnemosyne import cli`

(PLR0402)